lib/assets/javascripts/twine.coffee

```diff
@@ -69,11 +69,16 @@
       element = findOrCreateElementForNode(node)
       element.indexes = indexes
 
-    for type, binding of Twine.bindingTypes when definition = Twine.getAttribute(node, type)
-      element = findOrCreateElementForNode(node)
-      element.bindings ?= []
-      element.indexes ?= indexes
-
+    element = findOrCreateElementForNode(node)
```

Comment:
Also this will cause twine to take up way more memory than we currently do because we'd store bindings for every node in the DOM, rather than just those with bindings as we do now